source/ODApplication.cpp

```diff
@@ -49,6 +51,11 @@
 void ODApplication::startGame(boost::program_options::variables_map& options)
 {
     ResourceManager resMgr(options);
+
+    LogManager logMgr;
+    logMgr.addSink(std::shared_ptr<LogSink>(new LogSinkConsole()));
```

Comment:
It is was pretty widely done by a retired member @oyvindln. He was quite skilled with c++ so I guess we can rely on his work. IIRC, he used something like Utils::makeUniquePtr (or something like that).
However, I don't really care about using unique_ptr orshared_ptr but I think the last is misleading. In your case, the wanted behaviour clearly is to use only one instance and a unique_ptr seems more adapted. For my part, if I see a shared_ptr, I would expect it to be used somewhere else.